mirrord/cli/src/wizard.rs

```diff
@@ -0,0 +1,671 @@
+//! # mirrord Wizard (aka onboarding Wizard)
+//!
+//! The `wizard` module contains everything needed to run the `mirrord wizard` CLI command, with the
+//! exception of the frontend code. This lives in a different top-level directory called
+//! `wizard-frontend`. More documentation specific to the frontend exists in
+//! `wizard-frontend/README.md`.
+//!
+//! **The Wizard operates as follows**:
+//!
+//! 1. Build the TypeScript frontend and compress `wizard-frontend/dist` into a Gzipped file
+//! 2. Use the `include_bytes!` macro to make [`COMPRESSED_FRONTEND`] when compiled with the
+//!    `wizard` feature
+//! 3. When the command is run, the file is unzipped and the frontend served on localhost with axum
+//!    as a Single Page App
+//! 4. The wizard opens in the browser and the user can create and download a config file
+//!
+//! ### Working on the Backend
+//!
+//! Note: To compile the backend, you need to enable the `wizard` feature, for which you need the
+//! compressed frontend file. This file is in the `.gitignore` and should NOT be committed.
+//!
+//! *Example full build and run script*:
+//! ```bash
+//! ## Build the frontend.
+//! cd wizard-frontend/ || exit
+//! npm run build
+//! cd ..
+//!
+//! ## Zip frontend.
+//! tar czf wizard-frontend.tar.gz --directory=wizard-frontend/dist .
+//!
+//! ## If you are not changing the frontend, the above steps do not have to be repeated.
+//! ## Build mirrord w/ wizard feature.
+//! cargo build --manifest-path=./Cargo.toml -p mirrord --features wizard
+//!
+//! ## Run the wizard command with trace level TRACE.
+//! RUST_LOG=mirrord=trace ./target/debug/mirrord wizard
+//! ```
+//!
```

Comment:
These docs are a bit too much, you just need to say that the `wizard` is gated on the `--features wizard`, no need to tell people how to build and run it.

I would leave the docs for `mirrord wizard` to the `cli` docs we have. Like, have some docs for what the wizard does there, to be consistent with the other commands.